twake/backend/node/src/services/notifications/services/engine/processors/new-channel-message/index.ts

```diff
@@ -9,6 +9,11 @@ import {
 } from "../../../../entities";
 import { ChannelMemberNotificationLevel } from "../../../../../channels/types";
 import { MentionNotification } from "../../../../types";
+import { trackedEventBus } from "../../../../../../core/platform/framework/pubsub";
```

Comment:
`trackedEventBus` is not a good name. Name it like `localBus` (same for other files)